src/App.tsx

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+const containQuery = (text: string, query: string): boolean => {
+  return text.toLowerCase().includes(query.toLowerCase());
+};
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const { title, description } = movie;
+
+    return containQuery(title, query.trim())
+      || containQuery(description, query.trim());
```

Comment:
you can trim query in your containQuerie function, and not repeat here 2 times this method